android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -220,7 +220,45 @@ public static void openAssetDetailsActivity(Activity activity, String chainId,
         activity.startActivity(assetDetailIntent);
     }
 
-    public static String[] getNetworksList(Activity activity) {
+    public static EthereumChain[] getCustomNetworks(EthereumChain[] chains) {
+        List<EthereumChain> al = new ArrayList<EthereumChain>();
+        for (EthereumChain chain : chains) {
+            switch (chain.chainId) {
```

Comment:
not quite important, but it is possible to re-use `boolean isCustomNetwork` function here to skip custom networks

Reply:
good point